.coderabbit.yaml

```
@@ -0,0 +1,30 @@
+  enabled: true
+gitleaks:
+  enabled: true
+hadolint:
+  enabled: true
+
+chat:
+auto_reply: true
+
+knowledge_base:
+code_guidelines:
+enabled: true
```

YAMLlint (1.37.1)

[error] duplication of key "enabled" in mapping

(key-duplicates)